python/ray/_private/accelerators/nvidia_gpu.py

```diff
@@ -49,9 +54,47 @@ def get_current_node_num_accelerators() -> int:
             pynvml.nvmlInit()
         except pynvml.NVMLError:
             return 0  # pynvml init failed
+        driver_version = pynvml.nvmlSystemGetDriverVersion()
         device_count = pynvml.nvmlDeviceGetCount()
+        cuda_devices = []
+        mig_enabled = os.environ.get(
+            ray_constants.RAY_ENABLE_MIG_DETECTION_ENV_VAR, False
```

Comment:
What the default value should be: true or false. Is there a case where user enables MIG but doesn't want Ray to detect them? What will happen in that case?

Reply:
I changed the method using `env_bool` in ray constants which detect env variable when we start ray